source/main/gui/mygui/Attribute.h

```diff
@@ -33,7 +33,7 @@ along with Rigs of Rods.  If not, see <http://www.gnu.org/licenses/>.
 namespace attribute
 {
 
-	// ����� ������� ��� �������� ������ �� ������������ �������

```

Comment:
The encoding broke here, too. Even though the comments are in Russian they should be kept. Original encoding is `Windows-1251`, could you convert them to UTF-8 properly, please?
Same goes for the BaseLayout.h file.